openpype/pipeline/colorspace.py

```diff
@@ -237,12 +236,13 @@ def get_data_subprocess(config_path, data_type):
         return json.loads(return_json_data)
 
 
-def compatible_python():
-    """Only 3.9 or higher can directly use PyOpenColorIO in ocio_wrapper"""
-    compatible = False
-    if sys.version_info.major == 3 and sys.version_info.minor >= 9:
-        compatible = True
-    return compatible
+def compatibility_check():
+    """Making sure PyOpenColorIO is importable"""
+    try:
+        import PyOpenColorIO  # noqa: F401
+    except (ImportError, ModuleNotFoundError):
```

Comment:
It seems [ModuleNotFoundError is a subclass of ImportError](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) so we don't need it.
Which is nice because it also only [got added since Py3.6](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) which would break compatibility for older hosts. :)
```suggestion
    except ImportError:
```